android/src/main/java/com/sbugert/rnadmob/RNAdMobRewardedVideoAdModule.java

```diff
@@ -90,6 +90,11 @@ public void onRewardedVideoAdLeftApplication() {
     public void onRewardedVideoCompleted() {
         sendEvent(EVENT_VIDEO_COMPLETED, null);
     }
+    
+    @Override
+    public void onRewardedVideoCompleted() {
```

Comment:
This method is already implemented, so should not be added again.

Reply:
I think Npm version don't have this. I've faced this issue before but that was my quick workaround for this I'll FIx this.